Tests/DependencyInjection/Fixtures/Bundles/AnnotationsBundle/Entity/TestServiceGeneratedValueEntity.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Fixtures\Bundles\AnnotationsBundle\Entity;
+
+use Doctrine\Bundle\DoctrineBundle\Mapping\ServiceGeneratedValue;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class TestServiceGeneratedValueEntity
+{
+    /**
+     * @ORM\Id
```

Comment:
@nicolas-grekas This one is about `@var` being separated from other annotations I think. It can be fixed by `vendor/bin/phpcbf`